import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -0,0 +1,389 @@
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	changeExportedThrottlePolicyFile(t, args)
+	// Import Throttling Policy to env 2 to update
+	output, err := importThrottlePolicy(t, args)
+	assert.Contains(t, output, "Successfully updated")
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate  policies are not equal
+	validatePoliciesEquality(t, false, args, importedPolicy)
+}
+
+// ValidateThrottlePolicyImportUpdateConflict : Validates importing existing Throttling Policy to create conflict
+func ValidateThrottlePolicyImportUpdateConflict(t *testing.T, args *PolicyImportExportTestArgs, adminUsername, adminPassword, policyType string) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, _ := exportThrottlePolicy(t, policyName, args)
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[policyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	changeExportedThrottlePolicyFile(t, args)
+	// Import Throttling Policy to env 2 for update conflict
+	output, err := importThrottlePolicy(t, args)
+	assert.Error(t, err, "Importation conflict expected")
+	assert.Contains(t, output, "409", "Unexpected error code")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate policies are  equal
+	validatePoliciesEquality(t, true, args, importedPolicy)
```

Comment:
What are we validating here? This is a conflict situation that occurs due to a mismatch of policies, right? Do we need this equality validation?

Reply:
fixed with 53f8cc8609cb4711d71c229466ebbd0f9c8f10e5